src/server/services/parkeren/parkeren.ts

```diff
@@ -1,50 +1,72 @@
-import { FeatureToggle } from '../../../universal/config/feature-toggles';
+import FormData from 'form-data';
+
 import { apiSuccessResult } from '../../../universal/helpers/api';
 import { AuthProfileAndToken } from '../../auth/auth-types';
-import { DataRequestConfig } from '../../config/source-api';
-import { getFromEnv } from '../../helpers/env';
 import { getApiConfig } from '../../helpers/source-api-helpers';
 import { requestData } from '../../helpers/source-api-request';
+import { captureMessage } from '../monitoring';
 
-type ParkerenUrlTransformedResponse = {
-  isKnown: boolean;
-  url: string | null;
-};
+export async function fetchParkeren(
+  requestID: RequestID,
+  authProfileAndToken: AuthProfileAndToken
+) {
+  const [isKnown, url] = await Promise.all([
+    hasPermitsOrProducts(requestID, authProfileAndToken),
+    fetchSSOURL(requestID, authProfileAndToken),
+  ]);
+  return apiSuccessResult({
+    isKnown,
+    url,
```

Comment:
Het kan zijn dat iemand wel known is maar er geen SSO url is. Ik zou dan hier toch liever de url naar de frontoffice zien ipv `undefined` .  API-response-wise is dat netter. Anders vertrouw je, denk ik, teveel op de aanwezigheid van een url in de frontend.

Reply:
Discussie hierboven verplaatst